ReactWindows/ReactNative.Shared/UIManager/AccessibilityTrait.cs

```diff
@@ -0,0 +1,64 @@
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// NOTE: This enum is organized based on priority of these traits (0 is the lowest),
+    /// which can be assigned to an accessible object. On native, all traits are combined as
+    /// a list. On desktop, trait with the maximum value is picked. Whenever you are adding
+    /// a new trait add it in the right priority order in the list.
+    ///
+    /// This is copied from ReactXP definition of AccessibilityTrait enum.
+    /// </summary>
+    public enum AccessibilityTrait
```

Comment:
>AccessibilityTrait [](start = 16, length = 18)

I think this enum should only include values that are supported by react-native-windows. Additionally, we shouldn't have comments that mention the value works on iOS, Android, etc. This is react-native-windows so these details are irrelevant.

Reply:
good catch! removed all but supported values, removed invalid comments. will add ignoring of unsupported RXP values in RNW C# code same as ios and android do.

---
In reply to: [158560366](https://github.com/reseul/react-native-windows/pull/1#discussion_r158560366) [](ancestors = 158560366)